### ISSUE TITLE
PAC reporting examples: Round 4 publishing

### ISSUE THREAD

**Issue by llienfec** [fecgov/fec-cms#2492]
To help PACs understand how to report independent expenditures, we will draft and post reporting examples that will go with the nonconnected and SSF sections in Help for Candidates and Committees.

#1935 covers round 4 language; #2211 covers images, edits, and approvals.  This issue covers the work to create pages, get final approvals, and post examples.

- [x] Enter compiled edits into Wagtail
- [x]  Send examples to RAD manager review 
- [x] RAD approval
- [x] Publish examples
- [x] Create links on Reporting Examples list page
- [x] Add reporting example cards and links to content pages
- [x] Report out new reporting examples to RAD

**Comment by llienfec** [comment]
I made the wagtail pages for the following examples yesterday.  Today I did a quick quality check and sent 4 images to Jim for updates.  Jim already sent them back and the pages have been updated.

### Examples covered this round:
- 24-hour reports
- 48-hour reports
- Estimating independent expenditures
- Disseminating & paying for IEs in the same reporting period
- Paying for an IE first and disseminating in a later reporting period
- Disseminating an IE first and paying in a later reporting period
- Aggregating independent expenditures (example scenario)
- Debts owed by the committee (general example)
- Repaying independent expenditure debt
- Repaying operating expenditure debt (threw this one in so we can be finished w/ debts)

**Comment by llienfec** [comment]
I sent all of the PAC IE examples to RAD for review.

**Comment by llienfec** [comment]
@bmathesonFEC Quick status update: I made all of the requested changes to these examples in December and then wasn't able to update the `Filing x reports` pages.  That problem is fixed!

I went into the `Filing PAC reports` page this morning to add the IE and Debt sections along with the titles of the reporting examples.  They still need links (along with all the other unchecked box stuff above).

I think that some of these (or the form 5 examples) may belong on the SSF page in some form or fashion that @dorothyyeager can help with.

Also need to complete the Form 5 reporting example lists to finish up the work at #1934.  We closed it after we made all the changes in the reporting examples, but couldn't move forward with the section.

Happy to answer questions, just let me know.  Thanks for picking this up, everyone!!! :tada:
